scripts/cov.sh

```diff
@@ -7,6 +7,8 @@ go test -v -covermode=atomic -coverprofile=./cov/nats.out
 go test -v -covermode=atomic -coverprofile=./cov/builtin.out ./encoders/builtin
 go test -v -covermode=atomic -coverprofile=./cov/protobuf.out ./encoders/protobuf
 go test -v -covermode=atomic -coverprofile=./cov/test.out -coverpkg=github.com/nats-io/go-nats ./test
+go test -v -covermode=atomic -coverprofile=./cov/test.out -coverpkg=github.com/nats-io/go-nats/encoders/builtin ./test -run EncBuiltin
```

Comment:
Don't reuse `test.out` otherwise you lose output from previous run. User `test1.out`.